packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -160,27 +175,27 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           id={ id }
           tabIndex={ disabled ? -1 : 0 }
           aria-expanded={ isOpen ? 'true' : 'false' }
-          role='combobox'
-          aria-autocomplete='none'
-          aria-live={ disabled || isOpen ? 'off' : 'assertive' }
+          role={ ariaAttrs.role }
           aria-label={ ariaLabel }
           aria-describedby={ id + '-option' }
-          aria-activedescendant={ isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0 ? (this._id + '-list' + selectedIndices[0]) : null }
```

Comment:
This makes me seriously wonder why the multiselect dropdown in this version moves focus, but the single select uses `activedescendant`. I mean just... why didn't we pick one and stick to it? There's no benefit to one over the other that is different for single vs. multiple selection.

(This isn't about your PR at all, just looking into our old implementation boggled my mind 🤣😭)